CODEOWNERS

```diff
@@ -1,3 +1,9 @@
 * @DataDog/dd-trace-js
 /packages/dd-trace/src/appsec/ @DataDog/appsec-js
 /packages/dd-trace/test/appsec/ @DataDog/appsec-js
+/packages/datadog-plugin-*/ @Datadog/apm-framework-integrations-reviewers-js
+/packages/datadog-instrumentations/ @Datadog/apm-framework-integrations-reviewers-js
+/packages/ddtrace/src/plugins/ @Datadog/apm-framework-integrations-reviewers-js
+/packages/ddtrace/test/plugins/ @Datadog/apm-framework-integrations-reviewers-js
+/packages/ddtrace/src/service-naming/ @Datadog/apm-framework-integrations-reviewers-js
+/packages/ddtrace/test/service-naming/ @Datadog/apm-framework-integrations-reviewers-js
```

Comment:
I think we need to add _both_ teams `@DataDog/dd-trace-js` and `@Datadog/apm-framework-integrations-reviewers-js`, otherwise anyone would require _our_ approval to merge anything into those folders, and the JS team's approval alone would not be enough. At least this is how I understand it. If you are sure that this is not the case, feel free to ignore this message.

Ref: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#:~:text=%23%20In%20this%20example%2C%20any%20change%20inside%20the%20%60/scripts%60%20directory%0A%23%20will%20require%20approval%20from%20%40doctocat%20or%20%40octocat.%0A/scripts/%20%40doctocat%20%40octocat

Reply:
🤦  fixing